src/ducks/apps/Containers.jsx

```diff
@@ -1,9 +1,10 @@
 import { connect } from 'react-redux'
 
-import { fetchApps, fetchInstalledApps, uninstallApp, getInstalledApps, getRegistryApps, installAppFromRegistry } from './index'
+import { fetchApps, fetchInstalledApps, uninstallApp, getInstalledApps, getRegistryApps, installAppFromRegistry, installApp } from './index'
 
 import MyApplicationsComponent from './components/MyApplications'
 import DiscoverComponent from './components/Discover'
+import HiddenInstallerViewComponent from './components/HiddenInstallerView'
```

Comment:
I don't get the role of this component. Why is it hidden ?
I feel that `View` is redundant with `Component`. What do you think ?

Reply:
It's the previous hidden view of cozy-settings used to install any apps using a form.
I know about the `Component` naming part but it's just for this file since it will be exported as `HiddenInstallerView` as a connected component just below.

Reply:
Ah ok I see. I surely miss something, I did know that this view should come back. Is it because the stack registry is not ready yet ?

Reply:
For that and also because we could need to install an app not yet in the registry and the store won't allow it.